.taskcluster.yml

```diff
@@ -122,19 +122,19 @@ tasks:
                           source: '${repoUrl}/raw/${head_sha}/.taskcluster.yml'
                         - $if: 'tasks_for in ["github-push", "github-pull-request", "github-release"]'
                           then:
-                            name: "Decision Task"
+                            name: "Decision Task (${tasks_for[7:]})"  # strip out "github-" from tasks_for
```

Comment:
We may want github-push to include the short_head_ref in the name, but this can be a followup.